controllers/datamover.go

```diff
@@ -129,6 +144,11 @@ func (r *DPAReconciler) ReconcileDataMoverController(log logr.Logger) (bool, err
 			fmt.Sprintf("performed %s on datamover deployment %s/%s", op, dataMoverDeployment.Namespace, dataMoverDeployment.Name),
 		)
 	}
+
+	_, err = r.createResticSecret(&dpa)
```

Comment:
Maybe rename the function to `CreateDataMoverSecret` ? And Also, not add this here, instead add this in the main OADP Oeprator Batch Reconcile function ?